courses/fundamentals_of_ada/020_declarations.rst

```diff
@@ -298,9 +195,6 @@ Based Numeric Literals
 .. code:: Ada
 
    based_literal ::= base # based_number [.based_number] # [exp]
-   base ::= numeral
-   based_number ::= extended_digit {[underline] extended_digit}
```

Comment:
Up to you: you removed the definition of "based_number", so there's no clear indication that underscores are still allowed. I would recommend adding the "based_number" definition, changing "extended_digit" to "base_digit" and leaving out the definition of "base_digit"
Also, replace "[exp]" with the same text you used for "decimal literal"